### DISCUSSION TITLE
[2022/09/06] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#221] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/220 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by chloe463** [comment]
Astro - https://astro.build/
Islands architecture - https://www.patterns.dev/posts/islands-architecture/
Logger - https://codesandbox.io/s/logger-xvxf0n?file=/src/Logger.tsx

NextJS to Astro: more control = faster sites  - https://www.youtube.com/watch?v=PSzCtdM20Fc
Bun vs Node: Round 2 (feat. Astro, Deno, Fresh and more) - https://www.youtube.com/watch?v=qCX8rw4qOSA

**Comment by NotFounds** [comment]
https://vitest.dev/
